websites/E/EternalNetwork TM/presence.ts

```diff
@@ -124,7 +124,7 @@ presence.on("UpdateData", async () => {
 			}
 
 			if (pathname.includes("/adm/")) {
-				presenceData.state = "Using administrating power over the forum !";
+				presenceData.state = "Administrating the forum";
```

Comment:
Ditto

Reply:
Moderate or Manage seem logical

Reply:
Changed in commit 6310c13

![Proof](https://i.imgur.com/T1S8TPT.png)

Reply:
Message changed.